samples/modelbuilder/ObjectDetection_StopSigns/StopSignDetectionML.Model/ConsumeModel.cs

```diff
@@ -0,0 +1,46 @@
+// This file was auto-generated by ML.NET Model Builder. 
+
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.IO;
+using Microsoft.ML;
+using StopSignDetectionML.Model;
+using System.Net.Http;
+using Microsoft.ML.Trainers;
+using System.Web;
+using System.Net;
+
+namespace StopSignDetectionML.Model
+{
+    public class ConsumeModel
+    {
+        private static Lazy<PredictionEngine<ModelInput, ModelOutput>> PredictionEngine = new Lazy<PredictionEngine<ModelInput, ModelOutput>>(CreatePredictionEngine);
+
+        private static WebClient client = new WebClient();
```

Comment:
nit: this is now unused

Reply:
Fixed with https://github.com/dotnet/machinelearning-samples/pull/882/commits/63c54104961e38d70c93e59db6da9687feb9b568